### PR TITLE
Artifacts for CNI v1.14.1 and Nodeagent v1.0.2

### PR DESCRIPTION
**Pull request by jayanthvn**
**What type of PR is this?**
Upgrade node agent image to v1.0.2

**Which issue does this PR fix**:
This incorporates the race condition issue in SDK for generating access logs - https://github.com/aws/aws-network-policy-agent/pull/51 

**What does this PR do / Why do we need it**:
Upgrades Node agent image

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

Yes, CNI is just rebuild of image and node agent testing can be found here - https://github.com/aws/aws-network-policy-agent/pull/51

**Automation added to e2e**:

Yes

**Will this PR introduce any new dependencies?**:

N/A

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
 No

**Does this change require updates to the CNI daemonset config files to work?**:

No

**Does this PR introduce any user-facing change?**:

No
```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: config/master/aws-k8s-cni.yaml [hunk 1]
```diff
@@ -374,7 +374,7 @@ spec:
       hostNetwork: true
       initContainers:
       - name: aws-vpc-cni-init
-        image: "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.14.0"
+        image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.14.0-1-gd9b3668a-dirty
```
Comment (jayanthvn) [inline config/master/aws-k8s-cni.yaml:377]: Issue with the generate script, will update it manually..

**Review by jdn5126 — APPROVED** [review]
minor nit that we should update the chart READMEs as well @jayanthvn , but otherwise this is good to go

**Review by jaydeokar — DISMISSED** [review]
lgtm

**Comment by jayanthvn** [comment]
Updates readme..thanks.

**Review by jdn5126 — APPROVED** [review]

**Pull request merged**